test/imagefsinfo.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+# vim: set syntax=sh:
+
+load helpers
+
+function setup() {
+	setup_test
+	start_crio
+}
+
+function teardown() {
+	cleanup_test
+}
+
+@test "image fs info with default settings should return matching container_filesystem and image_filesystem" {
+	version=$(crictl --version)
+	echo "$version"
```

Comment:
To make sure that you get the correct binary, you can use `$CRICTL_BINARY` variable, or there is a function, a helper, called `crictl` too. Not sure if this would work for you?

The following is probably the same:

```suggestion
	crictl --version
```

That said, the version output won't be shown unless the test fails, I believe. You might need to save it to a file somewhere and then add `cat /tmp/debug.log` in the test runner script. Or something to that effect.

I wish there was a simple `log` or `debug` function available in bats.

Reply:
Do we need that `$version` at all?

Reply:
I'll just drop it.